unity/Assets/Maroon/reusablePrefabs/NewRuler/Scripts/RulerLogic.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using Maroon.Physics;
+using Maroon.Physics.CoordinateSystem;
+using UnityEngine;
+using Maroon.GlobalEntities;
+
+namespace Assets.Maroon.reusablePrefabs.NewRuler.Scripts
```

Comment:
please change the namespace of all of your Ruler scripts to "Maroon.Tools.Ruler"